Integrations/integration-Sandblast.yml

```diff
@@ -222,63 +222,96 @@ script:
     if (command === 'sandblast-upload' || command === 'sandblast-query' || command === 'sb-upload' || command === 'sb-query') {
         var uniqueKeys = ['md5', 'sha1', 'sha256'];
         var ec = {};
-        if (raw.Contents.response &&
-            raw.Contents.response.av &&
-            raw.Contents.response.av.malware_info &&
-            raw.Contents.response.av.malware_info.malware_type) {
+        if (contents.response &&
+            contents.response.av &&
+            contents.response.av.malware_info &&
+            contents.response.av.malware_info.malware_type) {
             var maliciousData = {
                 Malicious: {
                     Vendor: 'Sandblast',
                     Description: 'Sandblast found this file malicious',
-                    Confidence: raw.Contents.response.av.malware_info.confidence,
-                    MalwareFamily: raw.Contents.response.av.malware_info.malware_family,
-                    MalwareType: raw.Contents.response.av.malware_info.malware_type,
-                    Severity: raw.Contents.response.av.malware_info.severity,
-                    SignatureName: raw.Contents.response.av.malware_info.signature_name,
+                    Confidence: contents.response.av.malware_info.confidence,
+                    MalwareFamily: contents.response.av.malware_info.malware_family,
+                    MalwareType: contents.response.av.malware_info.malware_type,
+                    Severity: contents.response.av.malware_info.severity,
+                    SignatureName: contents.response.av.malware_info.signature_name,
                 }
             };
             for (var key in uniqueKeys) {
-                if (raw.Contents.response[uniqueKeys[key]]) {
-                    maliciousData[uniqueKeys[key].toUpperCase()] = raw.Contents.response[uniqueKeys[key]];
+                if (contents.response[uniqueKeys[key]]) {
+                    maliciousData[uniqueKeys[key].toUpperCase()] = contents.response[uniqueKeys[key]];
                 }
             }
+            var dbot_con = {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'SubType': 'av'};
```

Comment:
@mayagoldb question: are we adding file dot score only with md5? should we add it with other hashes if we have them?

Reply:
@yuvalbenshalom @amshamah419 If our indicator is sha1 or sha256 we should also add dbot score

Reply:
@amshamah419 lets take a look at it together and see if there is a need to add sha1/256